src/effects/builtin/reverb/reverbviewmodel.h

```diff
@@ -1,9 +1,40 @@
 #pragma once
 
+#include "../common/abstracteffectmodel.h"
+
 namespace au::effects {
-class ReverbViewModel
+class ReverbEffect;
+struct ReverbSettings;
+class ReverbViewModel : public AbstractEffectModel
 {
+    Q_OBJECT
+    Q_PROPERTY(QVariantList paramsList READ paramsList NOTIFY paramsListChanged FINAL)
+    Q_PROPERTY(bool wetOnly READ wetOnly WRITE setWetOnly NOTIFY wetOnlyChanged FINAL)
+
 public:
     ReverbViewModel();
+
+    QVariantList paramsList() const;
+
+    Q_INVOKABLE void init();
+    Q_INVOKABLE void setParam(const QString& key, double val);
+
+    bool wetOnly() const;
+    void setWetOnly(bool newWetOnly);
+
+signals:
+    void paramsListChanged();
+    void wetOnlyChanged();
+
+private:
+
+    ReverbEffect* effect() const;
+    const ReverbSettings& settings() const;
+    ReverbSettings& mutSettings();
```

Comment:
What does `mut` stand for?

Reply:
mut - mutable 

technically we can name both methods the same, we will be allowed to override the name.
But I usually prefer to name the constant method and the method for changing differently, so that it is clear where the value is simply read, and where it is intended to be changed

Reply:
I agree with that. I'd have preferred "mutableSettings" (It got me to wonder if it were related to muting), but it's not so bad.